### ISSUE TITLE
update keyword search to include individuals in responsibility statements?

### ISSUE THREAD

**Issue by millertg** [VandyVRC/srophe-eXist-app#137]
@wsalesky When updating the keyword search function, is would it be possible to also include searches of respStmt elements for particular teams or individuals who had worked on an entry?

**Comment by wsalesky** [comment]
Yes we can do that. Do you want these to just appear in the keyword search, or do you want a new field for the respStmt?

**Comment by millertg** [comment]
I think just appearing in the keyword search is fine
